### DISCUSSION TITLE
Newbie to docker is stuggling with passing commands

### DISCUSSION THREAD

**Discussion by zephiowl** [itzg/docker-minecraft-server#1157] [Q&A] [answered]
I managed to start the server fine; all seems good but once I tried to op myself I keep receiving this error; and would love some help 

 sudo docker exec -i zen_jepsen op xanthus58
rpc error: code = 2 desc = oci runtime error: exec failed: container_linux.go:247: starting container process caused "exec: \"op\": executable file not found in $PATH"

thanks a ton!

**Comment by itzg** [comment, marked as answer]
You're missing the `rcon-cli` argument which is the Linux command to run:

https://github.com/itzg/docker-minecraft-server#interacting-with-the-server

**Reply by zephiowl** [reply]
Just tried that and it didn't work causes a similar error 

sudo docker exec -i zen_jepsen rcon-cli say hi
rpc error: code = 2 desc = oci runtime error: exec failed: container_linux.go:247: starting container process caused "exec: \"rcon-cli\": executable file not found in $PATH"

**Reply by zephiowl** [reply]
I managed to manually op myself via the ops.json file but I still cant get the command passthrough to work.

**Reply by itzg** [reply]
Can you provide `docker container inspect zen_jepsen`?

**Reply by zephiowl** [reply]
omg
wait
no
i think i tried running it on my teamspeak server i was messing around with to get used to docker XD

**Reply by zephiowl** [reply]
it was the case ahh; thank you so much, i really gotta figure out how to give them custom names XD
